java/src/main/java/ai/rapids/cudf/ColumnView.java

```diff
@@ -5160,6 +5168,13 @@ public HostColumnVector copyToHost() {
     }
   }
 
+  /**
+   * Calculate the total space required to copy the data to the host.
+   */
+  public long getHostBytesRequired() {
```

Comment:
nit: why is the method name `getHostBytesRequired ` not indicative that it is returning padded memory? 
Is this an oversight or will that be redundant?

Reply:
Technically all of the versions of the APIs would be more accurate when padding the data to 64-bit boundaries. The memory allocation APIs that we use don't take any form of alignment. For C malloc says that the value returned will be aligned at least as much as `std::max_align_t`

https://en.cppreference.com/w/cpp/memory/c/malloc

If we really wanted to be accurate we should use this value when computing the alignment. But most of the time 64-bit is good enough. The GPU has similar requirements for alignment on memory allocations.

We have an existing API for returning GPU memory that matches exactly with what the buffers say their size is. Because of this I thought it best not to change it, but add a new API. For host memory there is no existing API, so I thought it better to just have it default to the more correct thing.

But if someone feels strongly about it I can change it.

Reply:
```
      assertEquals(24, v0.getDeviceMemorySize()); // (6*4B)
      assertEquals(64, v0.getHostBytesRequired()); // account for alignment padding
```
When I read those lines in the test I came to the code to read what is the difference, the doc didn't specify. It's your decision